data/ecosystems/f/flow-community.toml

```diff
@@ -5674,3 +5680,75 @@ url = "https://github.com/Zubainah/nitda-blockchain-scholarship"
 
 [[repo]]
 url = "https://github.com/zwht/connect_wallet_demo"
+
+[[repo]]
+url = "https://github.com/mikelord007/fluid-finance"
+
+[[repo]]
+url = "https://github.com/ahmadnurfadilah/promptchain"
+
+[[repo]]
+url = "https://github.com/AmaanSayyad/Side-B"
+
+[[repo]]
+url = "https://github.com/Demeter-Lab/SwiftPay-Payroll"
+
+[[repo]]
+url = "https://github.com/Demeter-Lab/SwiftPay-AI-Summary-Generator"
+
+[[repo]]
+url = "https://github.com/Demeter-Lab/Swiftpay-Frontend"
+
+[[repo]]
+url = "https://github.com/OpeOginni/flow-sign"
+
+[[repo]]
+url = "https://github.com/ayushojha20/FlowQL"
+
+[[repo]]
+url = "https://github.com/DappCoderr/Loot---A-Treasure-Game-on-Flow"
+
+[[repo]]
+url = "https://github.com/Ayushjain2205/data-royale-flow"
+
+[[repo]]
+url = "https://github.com/abu-hasib/lwb3app"
+
+[[repo]]
+url = "https://github.com/abu-hasib/lwb3hackbot"
```

Comment:
how do these 2 repos use Flow? Looked at the code but wasn't sure

Reply:
Yes they are, you can see Flow Cadence code in their source code.

These are repositories obtained from the Hackathon project list.